frontend/src/css/modules/landing.scss

```diff
@@ -107,11 +107,50 @@ section.bg-section {
 /*simple header*/
 
 .sim-header {
-    font-size: 18px;
-    top: 0;
-    left: 0;
-    z-index: 1001;
-    width: 100%;
+  font-size: 18px;
+  top: 0;
+  left: 0;
+  z-index: 999;
+  width: 100%;
+  position: relative;
+}
+.sim-header nav {
```

Comment:
In scss, for nested classes use this syntax
.a{
  .b{
     .c{
        }
    }
}
or take help for other files. @Obtin